### PR TITLE
Re-enable net461 functional tests.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
aspnet/Coherence-Signed#540

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
@Eilon @muratg I was going to merge this into preview2 so we could have valid net461 testing there. Assuming I need approval for that 👍

**Comment by mikeharder** [comment]
I don't think anyone else has been getting approval for Preview2 changes yet

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs [hunk 1]
```diff
@@ -91,7 +92,12 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains(expectedCompilationContent, content);
         }
 
-        [Fact]
+        [ConditionalFact]
+        // MVC Generates portable PDBs when compiling Razor views. NET461 does not understand portable PDBs
```
Comment (ryanbrandenburg) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Does the Coherence-Signed issue cover reverting this too or do we need a tracking issue?

Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: I reference this test in the MVC issue so once that closes out it should be a reminder to revert this.

**Pull request merged**